post_office/models.py

```diff
@@ -250,6 +270,9 @@ class Log(models.Model):
     )
     date = models.DateTimeField(auto_now_add=True)
     status = models.PositiveSmallIntegerField(_('Status'), choices=STATUS_CHOICES)
+    recipient_delivery_status = models.PositiveSmallIntegerField(
+        _('Recipient Delivery Status'), choices=RecipientDeliveryStatus.choices, blank=True, null=True
+    )
```

Comment:
These two fields can be combined into `status` field. A log can only have one status.